src/components/axes/axis.ts

```diff
@@ -188,7 +188,7 @@ module Plottable {
     }
 
     protected _generateTickMarkAttrHash(isEndTickMark = false) {
-      var tickMarkAttrHash = {
+      var tickMarkAttrHash: { [key: string]: number | ((d: any) => number) } = {
```

Comment:
Is there a way to avoid this union? Shouldn't the second argument be `[key: string]: (d:any) => number`?